modules/varscan/1.0/header_hg19.txt

```diff
@@ -0,0 +1,43 @@
+##fileformat=VCFv4.1
```

Comment:
We will need to run this on an hg38 genome and supply an equivalent file for hg38 bams. We will also need a way to choose the appropriate file. I'm sure @BrunoGrandePhD will have thoughts on how best to handle this.

Reply:
we could use the `op.switch_on_wildcard` function to pick the right header depending on the genome build

Reply:
Good idea

Reply:
are he headers also different for somatic vs. unpaired runs?

Reply:
Good point. It probably is at least slightly different. You'd have to run it and see what the header looks like